web/kratos/oidc-setup.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env sh
```

Comment:
It would be better to create a function that handles repetitive logic for checking environment variables, IMHO.

Reply:
Okay I'll try to do it.